format/pyrdp/pdu/client_data.go

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) 2022-2023 GoSecure Inc.
+package pyrdp
```

Comment:
Ok to name package same as directory name? and maybe rename to `pyrdp_pdu` etc at import in format/pyrdp/pyrdp.go?

Reply:
Done in 4f8a312